features/txt-parfmt-props.feature

```diff
@@ -4,6 +4,17 @@ Feature: Get or set paragraph formatting properties
   I need a ParagraphFormat object with read/write formatting properties
 
 
+  @wip
+  Scenario Outline: Get tab stops object
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops is a CT_TabStops object
+      
+    Examples:  values for paragraph_format.tab_stop elements 
+      | value |
+      | no    |
+      | some  |
+      
```

Comment:
Okay, this is still a lot of acceptance tests to stack up before any implementation, so I'm going to push through this first one just so we're not crossing objects. Then the rest should be okay in a single commit.

* This should be a Scenario: not a Scenario Outline:, because there's only one case ...
* The title should be 'Get tab stops'. object is implied in the title for all these and just adds noise to spell it out.
* the given is "Given a paragraph format", period. All paragraph format objects have a TabStops object, whether or not it contains any tab stops. And, we always start with the parent object as the given, not further up the hierarchy.
* the then clause is "Then paragraph_format.tab_stops is a TabStops object". CT_TabStops is part of the oxml/xmlchemy layer, not an API object.
* the examples go away because it's a scenario rather than a scenario outline